src/router/index.js

```diff
@@ -27,6 +28,12 @@ const Router = new VueRouter({
     {
       path: '/chat/:roomId',
       name: 'ChatView',
+      children: [
```

Comment:
May I know what this route is used for?

Reply:
at first I wanted to make it /chat/xx/share, but it was buggy (2 share modal appear every-time), so i used it to concatenate the share.vue into the chatView.vue